components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/UnsupportedAddonsAdapter.kt

```diff
@@ -28,9 +29,11 @@ class UnsupportedAddonsAdapter(
     private val unsupportedAddonsAdapterDelegate: UnsupportedAddonsAdapterDelegate,
     private val unsupportedAddons: List<Addon>
```

Comment:
nit: To avoid using `_` we could remove the `private` modifier from  the`unsupportedAddons`and use another name for the list.

```kotlin
class UnsupportedAddonsAdapter(
    private val addonManager: AddonManager,
    private val unsupportedAddonsAdapterDelegate: UnsupportedAddonsAdapterDelegate,
    val addons: List<Addon>
) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
    private val unsupportedAddons : MutableList<Addon> = addons.toMutableList()
```